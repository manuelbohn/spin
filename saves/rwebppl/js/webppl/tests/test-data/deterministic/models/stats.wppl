[
  listMean([1, 2, 6]),
  listVar([1, 4, 7]),
  listStdev([1, 2])
];
