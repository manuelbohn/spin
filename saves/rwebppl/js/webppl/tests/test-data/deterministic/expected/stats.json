{
  "result": [3, 6, 0.5]
}
